formant_ros2_adapter/scripts/components/subscriber/ingester.py

```diff
@@ -63,7 +65,8 @@ def ingest(
         msg_timestamp: int,
         tags: Dict,
     ):
-
+        self.ingestion_analytics.log_received_message(topic)
```

Comment:
Lets log the received message from the ROS code that actually receives the ROS message  BasicSubscriberCoodinater i think